### ISSUE TITLE
add unique string to YOUR REFERENCE CODE in Direct Bank Transfer

### ISSUE THREAD

**Issue by rudolfix** [Neufund/platform-frontend#1960]
Please refer to https://github.com/Neufund/platform-backend/issues/1222 for specification on reference name

What we need to do

1. generate 16 random bytes (UUID)
2. encode base64
3. add to payment details (reference) as REF xxxxxxxx, after NF but before G and WL

example `NF AGHGCmR3u2SuxdyNPIksxTyAhKM REF za7VbYcSQU2zRgGQXQAm G WL`

now it will not fit in one line in the modal... so we can have two lines for the reference

**Comment by huesforalice** [comment]
yeah I know, there just isn't enough space for the whole thing...
